PrintManual.cpp

```diff
@@ -0,0 +1,55 @@
+#include "PrintManual.h"
+#include <iostream>
+
+const int COLORCELLSPACES = 11;
+const int ONEDIGITSPACES = 11;
+const int TWODIGITSPACES = 10;
+const int MAX_COLOR_COMBINATIONS = TelCoColorCoder::numberOfMajorColors * TelCoColorCoder::numberOfMinorColors;
+std::string SEPARATOR = "==========================================";
+
+void getColorNameSizes(int& majorColorSize, int& minorColorSize, TelCoColorCoder::ColorPair& colorPair) {
```

Comment:
This is a testable function - you haven't written a test for it yet, but it would be easy to write one
Such a test would guarantee its functionality for yourself, also for people who maintain it in future.